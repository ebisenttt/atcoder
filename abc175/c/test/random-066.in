-615908624630716 931057614463692 81776755349020
