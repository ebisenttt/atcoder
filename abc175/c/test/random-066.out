38305418161444
